go/routers/app_multiplexer/tests/test_views.py

```diff
@@ -30,7 +30,11 @@ def test_show_stopped(self):
         router = response.context[0].get('router')
         self.assertEqual(router.name, u"myrouter")
         self.assertContains(response, rtr_helper.get_view_url('start'))
-        self.assertNotContains(response, rtr_helper.get_view_url('stop'))
+        self.assertContains(
+            response,
+            '<button class="btn btn-danger action" data-action="stop" ' +
+            ' data-url="%s" disabled>Deactivate</button>'
+            % rtr_helper.get_view_url('stop'), html=True)
```

Comment:
I don't think we can actually do much better without adding a whole lot of extra complexity that will probably be worse than the current fragility. Let's just go with this and worry about it next time we change this stuff.